src/include/limits_if.h

```diff
@@ -240,7 +240,7 @@ void update_soft_limits(server_info *, queue_info *, resource_resv *);
  *		the counts structure
  * @return	int
  */
-int find_preempt_bits(counts *, const char *, resource_resv *);
+int find_preempt_bits(counts *, std::string &, resource_resv *);
```

Comment:
doing this makes this a c++ header, and therefore you can remove the `#ifdef __cplusplus` and `extern "C" {` stuff, since it can no longer be imported by C.

Reply:
You might as well move this into the scheduler directory while you are at it.  The scheduler is the only one who uses it.

Reply:
done!